airflow/bin/cli.py

```diff
@@ -84,14 +84,15 @@ def trigger_dag(args):
     session = settings.Session()
     # TODO: verify dag_id
     execution_date = datetime.now()
+    run_id = args.run_id or "manual__{0}".format(datetime.datetime.utcnow().isoformat())
```

Comment:
did you test this? datetime came from a `from datetime import datetime`, so datetime.datetime won't work. I think you should just use `execution_date.isoformat()`